README.md

```diff
@@ -1,107 +1,45 @@
-# InSpec for VMware
+# InSpec PowerCLI
```

Comment:
Should we still call it inspec for vmware? PowerCli is one resource?

Reply:
I agree here, there are other plugins (PowervRA and PowerNSX for instance) that we can add here too.